src/actions/vm.js

```diff
@@ -345,3 +346,12 @@ export function changeVmIconById ({ vmId, iconId }) {
     },
   }
 }
+
+export function checkVMAviability ({ vmId }) {
```

Comment:
Can we follow the convention that only first letter in an abbreviation is capitalized? Like: `checkVmAvailability`.

We already use that in e.g. `VmDialog`, `VmDetail`, and others. Similarly we use `CHECK_VM_AVIABILITY`, not `CHECK_V_M_AVIABILITY`